cmd/operator/ocp_quickstart_guide/README.md

```diff
@@ -0,0 +1,104 @@
+
+## Minimum Hardware Requirements
+### Intel SGX Enabled Server
+- Third Generation Intel® Xeon® Scalable Platform, code-named “Ice Lake” or later
+- Configure BIOS using below details
+    ![SGX Server BIOS](images/SGX-BIOS.PNG)
+    [**Note:** The BIOS configuration shown above is just for the reference. Please contact your BIOS vendor for details]
+
+## Installation
+### Prerequisites
+- Make sure Red Hat OpenShift Cluster is ready to use and the developing machine is RHEL and `oc` command is installed and configured properly. Please note that the following operation is verified on Red Hat OpenShift Cluster 4.11 and working machine RHEL-8.6
+- Install the `oc` command to your development machine
+- Follow the [link](https://docs.openshift.com/container-platform/4.10/hardware_enablement/psap-node-feature-discovery-operator.html) to install **NFD operator** (if it's not already installed).   
+    **Note:** Please only install the NFD operator and use steps below to create the NodeFeatureDiscovery instance.  
+    - Create the NodeFeatureDiscovery instance  
+    ```
+    $ oc apply -f https://raw.githubusercontent.com/intel/intel-device-plugins-for-kubernetes/v0.24.0/deployments/nfd/overlays/node-feature-discovery/node-feature-discovery-openshift.yaml
+    ```
+    - Create the NodeFeatureRule instance  
+    ``` 
+    $ oc apply -f https://raw.githubusercontent.com/intel/intel-device-plugins-for-kubernetes/v0.24.0/deployments/nfd/overlays/node-feature-rules/node-feature-rules-openshift.yaml
+    ```
+
+### Install Operator using OpenShift Web Console
+1.  In the OpenShift web console navigate to **Operator** -> **OperatorHub**
+2.  Search for **Intel Device Plugins Operator ->** Click **Install**  
+<img src="images/operator.PNG" width="300" height="200">
+
+### Verify Operator installation
+1.  Go to **Operator** -> **Installed Operators**
+2.  Verify the status of operator as **Succeeded**
+3.  Click **Intel Device Plugins Operator** to view the details  
+    ![Verify Operator](images/verify-operator.PNG)
+
+
+## Deploying Plugins
+The [`v0.24.0`](https://github.com/intel/intel-device-plugins-for-kubernetes/releases/tag/v0.24.0) release of the operator supports only the Intel SGX device plugin. GPU, Intel QAT, Intel DSA, Intel  IAA, and other device plugins will be supported in future releases.
```

Comment:
this repeats what was written in the intro so maybe remove?

Reply:
Done